src/algokit/core/proc.py

```diff
@@ -86,7 +86,7 @@ def run_interactive(
     command_str = " ".join(command)
     logger.debug(f"Running '{command_str}' in '{cwd or Path.cwd()}'")
 
-    result = subprocess_run(command, cwd=cwd, env=env)
+    result = subprocess_run(command, cwd=cwd, env=env, check=True)
```

Comment:
this should't be `check=True` that will cause an exception, the result code is being checked explicitly below instead.
did it hit a linting rule? if so, can explicitly specify default of `check=False`

Reply:
It did, cool. I will set to False.

Reply:
@negar-abbasi This seems to be still be `check=True`